nixos/modules/services/monitoring/prometheus/exporters/varnish.nix

```diff
@@ -68,7 +68,6 @@ in
   serviceOpts = {
     path = [ pkgs.varnish ];
     serviceConfig = {
-      DynamicUser = true;
```

Comment:
I'm curious, does DynamicUser not work here?

Reply:
It needs to access /var/spool/varnish. Running as dynamic user, the exporter is unable to find varnish.

Reply:
Makes sense, thanks.